### PR TITLE
feat: convert basic css tests to Playwright

### PR DESCRIPTION
**Pull request by Sembauke**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

File: cypress/e2e/default/learn/responsive-web-design/show-cert-from-superblock.ts
Comment (naomi-lgbt) [inline cypress/e2e/default/learn/responsive-web-design/show-cert-from-superblock.ts:1]: Should this file be deleted in this PR? Seems unrelated to the tests that were migrated.

Comment (Sembauke) [reply cypress/e2e/default/learn/responsive-web-design/show-cert-from-superblock.ts:1]: Seems I have mixed things up a bit.

**Review by naomi-lgbt — APPROVED** [review]

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:
